### DISCUSSION TITLE
There should be a separate discussion on English as well

### DISCUSSION THREAD

**Discussion by sobhardwaj** [umijs/umi#10924] [General]
UMI is absolutely great.

**Comment by akmalelias** [comment]
Hi, I'm wondering if there is any intention to translate the UmiJS website and documentation into English?

**Reply by skitsanos** [reply]
Oh, you are not alone in this boat :) I came from Umi 3.x, and even if I some things can do by 'heart' sometimes there is still a need to check something with the manual...

**Reply by Douglas0n** [reply]
Hi, @skitsanos, Actually its one of the items in 2023 Roadmap: https://github.com/umijs/umi/issues/10423 - （文档）Umi 中文到英文翻译脚本，diff 精确到行 @sorrycc
(Documentation - Chinese to English script, accurate line diff). 

The framework is great! We're using UmiJS around here and I know how important the documentation is.
I have published a book in collaboration with Packet covering UmiJS v3, if you are interested: https://amzn.to/3KVYmxh

**Reply by skitsanos** [reply]
Oh, this is nice :) Congratulations on the book!

Personally, I am already on v.4 since it just appeared, knowing Umi like a 'local' :) Literally every single project is on Umi. Just for the SSR, the v4 is still not there yet, so for the SSR I use Ice.js for now...
